docs/tutorials/in-the-cloud.qmd

```diff
@@ -0,0 +1,302 @@
+---
+title: Working in the cloud
+description: >
+    Get you and your collaborators performing linkage in the cloud
+---
+
+This tutorial provides an overview of how to use `pprl_toolkit` on
+Google Cloud Platform (GCP). We go over how to assemble and assign roles in a
+linkage team, how to set up everybody's projects, and end with executing the
+linkage itself.
+
+![A diagram of the PPRL cloud architecture, with the secure enclave and key management services](https://github.com/datasciencecampus/pprl_toolkit/blob/main/assets/pprl_cloud_diagram.png?raw=true)
+
+Above is a diagram showing the PPRL cloud architecture. The cloud demo uses a
+Google Cloud Platform (GCP) Confidential Space compute instance, which is a
+virtual machine (VM) using AMD
+[Secure Encrypted Virtualisation](https://www.amd.com/en/developer/sev.html)
+(AMD-SEV) technology to encrypt data in-memory. The Confidential Space VM can
+also provide cryptographically signed documents, called attestations, which the
+server can use to prove that it is running in a secure environment before
+gaining access to data.
+
+
+## Assembling a linkage team
+
+There are four roles to fill in any PPRL project: two data-owning **parties**,
+a workload **author**, and a workload **operator**. A workload is how we refer
+to the resources for the linkage operation itself (i.e. the containerised
+linkage code and the environment in which to run it.)
+
+These roles need not be fulfilled by four separate people. It is perfectly
+possible to perform PPRL on your own, or perhaps you are working under a trust
+model that allows one of the data-owning parties to author the workload while
+the other is the operator.
+
+::: {.callout-tip}
+In fact, `pprl_toolkit` is set up to allow any configuration of these roles
+among up to four people.
+:::
+
+In any case, you must decide who will be doing what from the outset. Each role
+comes with different responsibilities, but all roles require a GCP account and
+access to the `gcloud` command-line tool. Additionally, everyone in the linkage
+project will need to install `pprl_toolkit`.
+
+### Data-owning party
+
+Often referred to as just a **party**, a data owner is responsible for the
```

Comment:
We should say here that the data-owning parties also each set up a storage bucket and a key management server, and a workload identity pool that allows them to share permissions with the server that conducts the matching.

Reply:
Agreed. I am signing off now, but I will pick this up tomorrow.

Reply:
Fixed by d0ee3da